packages/react-app/src/components/ThemeSwitch.jsx

```diff
@@ -5,6 +5,7 @@ import { useThemeSwitcher } from "react-css-theme-switcher";
 export default function ThemeSwitcher() {
   const theme = window.localStorage.getItem("theme");
   const [isDarkMode, setIsDarkMode] = useState(!(!theme || theme === "light"));
+  // eslint-disable-next-line
```

Comment:
Instead of `eslint-disable-next-line` you can remove the status variable since the warning is `Line 8:35:  'status' is assigned a value but never used  no-unused-vars`.